packages/app-desktop/ElectronAppWrapper.ts

```diff
@@ -462,6 +474,14 @@ export default class ElectronAppWrapper {
 		this.customProtocolHandler_ ??= handleCustomProtocols(logger);
 	}
 
+	// eslint-disable-next-line @typescript-eslint/no-explicit-any
+	public initializeAutoUpdaterService(logger: LoggerWrapper, initializedShim: any, devMode: boolean, allowPrereleaseUpdates: boolean) {
```

Comment:
Small thing, but could you please use the same name as the setting variable for the prerelease boolean. So `includePreReleases`